### PR TITLE
Numpy casting error - again

### PR DESCRIPTION
**Pull request by josenavas**
In the last refactor, I forgot to cast numpy values to python values (again) so we're hitting the can't adopt type np.bool_ error again. I added a specific test so we don't hit this error again.

### CODE REVIEW

**Comment by antgonza** [comment]
:+1:

**Comment by wasade** [comment]
np.asscalar returns a python type? Pretty certain those are scalar numpy
types

**Comment by josenavas** [comment]
It returns python types. In the tests, I'm specifically testing that the returned values are not an instance of `np.generic`

**Comment by josenavas** [comment]
BTW, looks like Travis is not running?

**Comment by wasade** [comment]
Crazy...

**Comment by ElDeveloper** [comment]
@josenavas yeah I don't know what might be going on, would you mind re-opening this PR? Or maybe pushing another commit?

File: qiita_db/metadata_template/util.py [hunk 1]
```diff
@@ -68,6 +68,24 @@ def get_datatypes(metadata_map):
     return [type_lookup(dtype) for dtype in metadata_map.dtypes]
 
 
+def cast_to_python(value):
+    """Casts the value form numpy types to python types
```
Comment (ElDeveloper) [inline qiita_db/metadata_template/util.py]: from -> from

File: qiita_db/metadata_template/test/test_util.py [hunk 2]
```diff
@@ -48,6 +51,20 @@ def test_get_datatypes(self):
         exp = ['float8', 'varchar', 'integer']
         self.assertEqual(obs, exp)
 
+    def test_cast_to_python(self):
+        """Correctly returns the value casted"""
+        b = np.bool_(True)
+        obs = cast_to_python(b)
+        self.assertTrue(obs)
+        self.assertFalse(isinstance(obs, np.bool_))
```
Comment (ElDeveloper) [inline qiita_db/metadata_template/test/test_util.py]: Wouldn't it be better to assert that the instance is indeed a 'bool' type?

Comment (josenavas) [inline qiita_db/metadata_template/test/test_util.py]: Done

File: qiita_db/metadata_template/util.py (hunk 1 above)
Comment (josenavas) [inline qiita_db/metadata_template/util.py]: Done

**Comment by josenavas** [comment]
Thanks @ElDeveloper I addressed your comments and now Travis decided to test the branch :smile:

**Comment by ElDeveloper** [comment]
Thanks @josenavas

**Pull request merged**